src/global/helpers/date-helper.ts

```diff
@@ -34,3 +42,35 @@ export function removeTimezoneFromISOTimeString(isoTime: string): Date | undefin
   }
   return new Date(isoTime);
 }
+
+/**
+ * This function converts a duration number to date string with specific format
+ *
+ * @param duration - duration in minutes
+ * @returns a new date string in the format "dd hh mm"
+ * */
+
+export const durationToTime = (duration: number): string => {
+  const result = [];
+  const now = 0;
+  let future = addMinutes(now, duration);
+
+  const days = differenceInDays(future, now);
+  if (days > 0) {
+    result.push(`${days} d`);
```

Comment:
Is this really intended, that there is no i18n now?

Reply:
Yes, sbb mobile solve this like that aswell.